packages/lesswrong/components/sequences/LargeSequencesItem.tsx

```diff
@@ -123,53 +134,60 @@ export const LargeSequencesItem = ({sequence, showAuthor=false, classes}: {
   showAuthor?: boolean,
   classes: ClassesType,
 }) => {
-  const { UsersName, ContentStyles, SequencesSmallPostLink, ContentItemTruncated } = Components
+  const { UsersName, ContentStyles, SequencesSmallPostLink, ContentItemTruncated, LWTooltip } = Components
   
   const [expanded, setExpanded] = useState<boolean>(false)
 
   const cloudinaryCloudName = cloudinaryCloudNameSetting.get()
 
+  let posts : PostsList[] = []
+  sequence.chapters?.forEach(chapter => chapter.posts?.forEach(post => posts.push(post)))
```

Comment:
can be `const posts = sequence.chapters?.flatMap(chapter => chapter.posts ?? [])`, I think?